### PR TITLE
feat:added an empty page with the route: /settings/accounts/new.

### PR DESCRIPTION
**Pull request by lakshay-saini-au8**
Added Empty page with the route as /settings/accounts/new

Screenshot 2023-12-12 at 9 19 51 PM

Added Storybook story for SettingsNewAccount Components
Screenshot 2023-12-13 at 8 47 08 PM

### CODE REVIEW

**Comment by lakshay-saini-au8** [comment]
@thaisguigon 
Please Review

File: packages/twenty-front/src/pages/settings/accounts/SettingsNewAccount.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import { SettingsPageContainer } from '@/settings/components/SettingsPageContainer';
+import { IconSettings } from '@/ui/display/icon';
+import { SubMenuTopBarContainer } from '@/ui/layout/page/SubMenuTopBarContainer';
+import { Breadcrumb } from '@/ui/navigation/bread-crumb/components/Breadcrumb';
+
+export const SettingsNewAccount = () => {
+  return (
+    <SubMenuTopBarContainer Icon={IconSettings} title="Settings">
+      <SettingsPageContainer>
+        <SettingsHeaderContainer>
```
Comment (thaisguigon) [inline packages/twenty-front/src/pages/settings/accounts/SettingsNewAccount.tsx:11]: `SettingsHeaderContainer` is not needed. We created it for when a button is needed besides the Breadcrumb, which is not the case for this page.

File: packages/twenty-front/src/pages/settings/accounts/SettingsNewAccount.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { SettingsHeaderContainer } from '@/settings/components/SettingsHeaderContainer';
+import { SettingsPageContainer } from '@/settings/components/SettingsPageContainer';
+import { IconSettings } from '@/ui/display/icon';
+import { SubMenuTopBarContainer } from '@/ui/layout/page/SubMenuTopBarContainer';
+import { Breadcrumb } from '@/ui/navigation/bread-crumb/components/Breadcrumb';
+
+export const SettingsNewAccount = () => {
```
Comment (thaisguigon) [inline packages/twenty-front/src/pages/settings/accounts/SettingsNewAccount.tsx:7]: Could you add a Storybook story for this component?

**Review by thaisguigon — CHANGES_REQUESTED** [review]
Thank you for your contribution @lakshay-saini-au8!
A few minor comments, otherwise LGTM!

File: packages/twenty-front/src/pages/settings/accounts/SettingsNewAccount.tsx (hunk 1 above)
Comment (lakshay-saini-au8) [reply packages/twenty-front/src/pages/settings/accounts/SettingsNewAccount.tsx]: Got it

**Comment by lakshay-saini-au8** [comment]
@thaisguigon 
Please review it 
I updated the PR

**Review by thaisguigon — APPROVED** [review]
Thank you! I added a commit to fix the files linting.

**Pull request merged**